frontend/viewer/src/lib/components/ui/alert-dialog/alert-dialog-action.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+  import {buttonVariants} from '$lib/components/ui/button/index.js';
+  import {cn} from '$lib/utils.js';
```

Comment:
I wonder why these were imported as js files since they're ts. I guess it doesn't really matter since it works

Reply:
But we should change it I guess.